lib/rentlinx/models/base.rb

```diff
@@ -44,6 +40,10 @@ def get_units_for_property_id(id)
       Rentlinx.client.get_units_for_property_id(id)
     end
 
+    def get_photos_for_property_id(id)
```

Comment:
I think at first the motivation was to keep all the calls to `Rentlinx.client` in the base file, but `Photoable` started breaking this and I agree with you, it makes more sense to move these methods.